pzsd_bot/ui/triggers/modals.py

```diff
@@ -22,6 +29,19 @@ def is_valid_regex(pattern: str) -> bool:
         else:
             return True
 
+    def is_valid_emoji(self, emoji_str: str) -> bool:
+        if emoji.is_emoji(emoji_str):
+            return True
+
+        if emoji.emojize(emoji_str, language="alias") != emoji_str:
+            return True
```

Comment:
I actually think this might be redundant now that I `emojize` all the responses on L68 prior to doing this check.